src/features/DebugSession.ts

```diff
@@ -734,28 +745,26 @@ export class PickRunspaceFeature extends LanguageClientConsumer {
                             this.clearWaitingToken();
                             reject();
 
-                            void this.logger.writeAndShowError("Attach to PowerShell host process: PowerShell session took too long to start.");
+                            void this.logger.writeAndShowError(
+                                "Attach to PowerShell host process: PowerShell session took too long to start.");
                         }
                     }, 60000);
                 },
             );
         }
     }
 
-    private async pickRunspace(processId: string): Promise<string | undefined> {
+    private async pickRunspace(processId: number): Promise<number | undefined> {
+        // We need the language client in order to send the request.
+        await this.getLanguageClient();
+
         const response = await this.languageClient?.sendRequest(GetRunspaceRequestType, { processId });
         const items: IRunspaceItem[] = [];
         for (const runspace of response ?? []) {
-            // Skip default runspace
-            if ((runspace.id === 1 || runspace.name === "PSAttachRunspace")
```

Comment:
We do need to keep the PSAttachRunspace (it's the bridge, we can't debug it). -- Patrick